README.md

```diff
@@ -1,3 +1,11 @@
 This is a toy raytracer I wrote in rust to learn how raytracers work. I also
 wrote a series of posts on it starting
 [here](https://bheisler.github.io/post/writing-raytracer-in-rust/).
+
+To run the example, clone the repository and
+
+```
+cd raytracer
+cd app
+cargo run scenes/test.json scenes/checkerboard.png
```

Comment:
The second argument is the output file. This will overwrite the checkerboard texture file, which is probably not what you intended.